RealEstate/apps/core/views.py

```diff
@@ -134,37 +133,15 @@ def post(self, request, *args, **kwargs):
         """
         homebuyer = Homebuyer.objects.filter(user_id=request.user.id)
         couple = Couple.objects.filter(homebuyer__user=request.user)
-        categories = Category.objects.filter(couple=couple)
         house = get_object_or_404(House.objects.filter(id=kwargs["house_id"]))
 
-        default_score = Grade._meta.get_field('score').default
-        for category in categories:
-            value = request.POST.get(str(category.id)) or default_score
-            grade, created = Grade.objects.update_or_create(
-                homebuyer=homebuyer.first(), category=category, house=house, defaults={'score': int(value)})
-
-        grades = Grade.objects.filter(house=house, homebuyer=homebuyer)
-
-        # Merging grades and categories to provide object with both information.
-        # Data Structure: [(cat1, score1), (cat2, score2), ...]
-        graded = []
-        for category in categories:
-            missing = True
-            for grade in grades:
-                if grade.category.id is category.id:
-                    graded.append((category, grade.score))
-                    missing = False
-                    break
-            if missing:
-                graded.append((category, None))
-        form = EvaluationForm(graded=graded)
-
-        messages.success(request,"Your evaluation was saved!")
-        context = {
-            'couple': couple,
-            'house' : house,
-            'grades': graded,
-            'form' : form
-        }
-        context.update(self._score_context())
-        return render(request, self.template_name, context)
+        if request.is_ajax():
+          id = request.POST['category']
+          score = request.POST['score']
```

Comment:
So this issue existed from the previous branch, but I had already made a ton of comments and didnt want to overload the pull request, however I should mention it now that there are no forms remaining in the view.  The form instances are useful for two reasons:
- Knowing how to draw itself in HTML with the correct widgets
- Knowing how to validate data that is entered from users.

The second piece is what is missing here.  Usually the pattern is:
```python
form = SomeForm(request.POST)
if form.is_valid():
   make_changes_to_db()
   redirect
rerender_with_errors()
```

The id and score variables are being read directly from the request.POST data.  If the user goes through our UI, its fine.  But if they hack together a POST request with malicious data, they could create Grades for categories that are not theirs, or give themselves very high scores.  Ideally there should be some kind of check to make sure the score is within a valid range, and the category is valid for the requesting user.  However this might be something we have to weigh with time constraints in the interest of finishing other features first.